build_tools/pkgci/README.md

```diff
@@ -0,0 +1,12 @@
+# PkgCI Scripts
+
+This directory contains scripts and configuration for the "new" CI, which
+is based on building packages and then flowing those to followon jobs.
+
+The traditional CI attempted to do all steps as various kinds of source
+builds at head vs a split package/test style of workflow. It can mostly
+be found in the `cmake` directory but is also scattered around.
```

Comment:
We also have the [`build_tools/github_actions/`](https://github.com/openxla/iree/tree/main/build_tools/github_actions) directory, but it looks like these scripts focus more on how the builds are configured than how we interface with the CI system itself.

Reply:
I'm not planning to integrate any of this with the existing directory structure/tools, which I very much consider to be "v1" stuff. I'd like a clean break here with a focus on hermetic scripts that run locally/on-CI as expected.